src/Aspire.Dashboard/Extensions/TelemetryFilterFormatter.cs

```diff
@@ -23,7 +23,9 @@ private static string SerializeFilterToString(TelemetryFilter filter)
             _ => null
         };
 
-        return $"{Escape(filter.Field)}:{condition}:{Escape(filter.Value)}";
+        var state = filter.Enabled ? "enabled" : "disabled";
+
+        return $"{Escape(filter.Field)}:{condition}:{Escape(filter.Value)}:{state}";
```

Comment:
Could you make it so the `:disabled` is only added when disabled? Most filters will always be enabled. Also, keeps compatibility with bookmarks of existing filters